python/dgl/distributed/nn/pytorch/sparse_emb.py

```diff
@@ -16,11 +16,11 @@ class DistEmbedding:
 
     To support efficient training on a graph with many nodes, the embeddings support sparse
     updates. That is, only the embeddings involved in a mini-batch computation are updated.
-    Currently, DGL provides only one optimizer: `SparseAdagrad`. DGL will provide more
-    optimizers in the future.
+    Currently, DGL provides two optimizer: :class:`dgl.distributed.optim.SparseAdagrad` and
+    :class:`dgl.distributed.optim.SparseAdam`. DGL will provide more optimizers in the future.
```

Comment:
Let's not explicitly list the optimizers here to avoid frequent update in the future. Put a link to https://docs.dgl.ai/api/python/dgl.distributed.html#distributed-embedding-optimizer .